debian/cloudstack-agent.install

```diff
@@ -18,12 +18,11 @@
 /etc/cloudstack/agent/agent.properties
 /etc/cloudstack/agent/environment.properties
 /etc/cloudstack/agent/log4j-cloud.xml
+/etc/default/cloudstack-agent
 /etc/profile.d/cloudstack-agent-profile.sh
 /etc/logrotate.d/cloudstack-agent
-/etc/init.d/cloudstack-agent
 /usr/bin/cloudstack-setup-agent
```

Comment:
The built deb package still included and install cloudstack-agent initd script at /etc/init.d/cloudstack-agent and same thing happened with cloudstack-usage script. For running them with systemd, are they required? Removing them from *.install file still included them in the package.

dpkg -c <deb file>
```
...
drwxr-xr-x root/root         0 2016-05-13 10:00 ./etc/cloudstack/
drwxr-xr-x root/root         0 2016-05-13 10:00 ./etc/cloudstack/agent/
-rwxr-xr-x root/root      6529 2016-05-13 10:00 ./etc/cloudstack/agent/agent.properties
-rwxr-xr-x root/root       906 2016-05-13 10:00 ./etc/cloudstack/agent/environment.properties
-rwxr-xr-x root/root      3543 2016-05-13 10:00 ./etc/cloudstack/agent/log4j-cloud.xml
drwxr-xr-x root/root         0 2016-05-13 10:00 ./etc/init.d/
-rwxr-xr-x root/root      4993 2016-05-13 10:00 ./etc/init.d/cloudstack-agent
```